agent/agent.go

```diff
@@ -372,7 +354,7 @@ func (a *Agent) init() error {
 }
 
 func (a *Agent) installSignalHandlers() {
-	signal.Reset(syscall.SIGINT, syscall.SIGTERM, syscall.SIGUSR1, syscall.SIGUSR2, syscall.SIGHUP)
+	signal.Reset(syscall.SIGINT, syscall.SIGTERM, syscall.SIGHUP)
```

Comment:
what is the side effect of this?  SIGUSR1 & SIGUSR2 aren't supported on windows...do we need them there?